criu-3.15/lib/py/cli.py

```diff
@@ -1,19 +1,24 @@
 from __future__ import print_function
 import struct
 from jsonpath_ng import parse
+from elftools.elf.elffile import ELFFile
```

Comment:
Can all the API requests from ELF tools be moved to the file elf_utils.py?

Reply:
Based on the priority of this change request, it can be considered. The change would require re-testing both transform and stack-shuffle.

Reply:
We just need to move the api usage into another file, why would it need re-testing?